src/boot/O2/mtxuty-cvt.c

```diff
@@ -7,9 +7,13 @@ void MtxConv_F2L(Mtx* mtx, MtxF* mf) {
     for (i = 0; i < 4; i++) {
         for (j = 0; j < 4; j++) {
             s32 value = (mf->mf[i][j] * 0x10000);
+            struct {
+                u16 intPart[4][4];
+                u16 fracPart[4][4];
+            }* mu = (void*)mtx;
```

Comment:
How come you define this struct internally in the function instead of externally? (and in su_mtx.c)

Reply:
That is just what we did in AF (since libultra doesn't have the union). I'm not completely opposed to having it external somewhere. The question becomes what to name it and where to put it.

Reply:
I'm not too worries about it, if it's for consistency then I'm fine with how it is